hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -112,17 +114,20 @@ public class TokenClaimAirdropTest extends TokenAirdropBase {
 
     @BeforeAll
     static void beforeAll(@NonNull final TestLifecycle lifecycle) {
+        // override and preserve old values
         lifecycle.overrideInClass(Map.of(
                 "entities.unlimitedAutoAssociationsEnabled", "false",
                 "tokens.airdrops.enabled", "false",
-                "tokens.airdrops.claim.enabled", "false"));
+                "tokens.airdrops.claim.enabled", "false",
+                "tokens.airdrops.cancel.enabled", "false"));
         // create some entities with disabled airdrops
         lifecycle.doAdhoc(setUpEntitiesPreHIP904());
         // enable airdrops
         lifecycle.doAdhoc(
                 overriding("entities.unlimitedAutoAssociationsEnabled", "true"),
                 overriding("tokens.airdrops.enabled", "true"),
-                overriding("tokens.airdrops.claim.enabled", "true"));
+                overriding("tokens.airdrops.claim.enabled", "true"),
+                overriding("tokens.airdrops.cancel.enabled", "true"));
```

Comment:
Above overrideInClass() call is all the default TokesConfig values. EntitiesConfig also has unlimitedAutoAssociationsEnabled default to true. I think we only need lifecycle.overrideInClass() for "tokens.airdrops.enabled", "true", "tokens.airdrops.claim.enabled", "true","tokens.airdrops.cancel.enabled", "true" here. 
               @tinker-michaelj is the above correct?

Reply:
Here, we override airdrop related properties twice. 
1 - we disabled everything using `overrideInClass`.
This way we can create entities like if they are created pre hip904 in `setUpEntitiesPreHIP904`
For example, a hollow account with 0 auto-associations.
2 - After that we enable again everything related to the airdrops, and then start tests.

Reply:
We only need  ```
lifecycle.overrideInClass(Map.of(
                "entities.unlimitedAutoAssociationsEnabled", "false"))
``` instead of 
                l```
ifecycle.overrideInClass(Map.of(
                "entities.unlimitedAutoAssociationsEnabled", "false",
                "tokens.airdrops.enabled", "false",
                "tokens.airdrops.claim.enabled", "false")); 
```since the other two are the default config value.

Reply:
But when we release the airdrops, the default values will be changed to "true". 
Also, in different environments, we can have different values. 
I don't think we should be relying on default values here.

Reply:
I see. Thanks.